packager/react-packager/src/Resolver/polyfills/prelude.js

```diff
@@ -1,5 +1,4 @@
-/* eslint global-strict:0 */
-__DEV__ = false;
+/* eslint strict:0 */
+global.__DEV__ = false;
```

Comment:
cc @cpojer 

@vjeux we have the WPO transform that reads it – https://github.com/facebook/react-native/blob/f546dc3790edd4e0b310caccdc273f31a717258c/packager/react-packager/src/transforms/whole-program-optimisations/dead-module-elimination.js#L63-L76
It will break.